### DISCUSSION TITLE
Adobe position

### DISCUSSION THREAD

**Discussion by bramstein** [miguelsousa/openbakery#22] [General]
:wave: I'm just curious if Adobe Fonts plans to use OpenBakery instead of FontBakery for its font processing pipeline. We (The Type Founders) specifically started using FontBakery for validating our fonts to streamline onboarding fonts on Adobe Fonts. Does this fork imply that we'll need to run our fonts through OpenBakery in the (near?) future? This could be especially important if checks & profiles start diverging from the main FontBakery project, or will the Adobe Fonts profile in FontBakery also be maintained by Adobe?

**Comment by miguelsousa** [comment]
Hi Bram. Thanks for asking these questions. I imagine other foundry partners may start wondering about these things as well. We haven't made any announcements yet because OpenBakery still needs a bit more work before we consider it ready for general use.

> I'm just curious if Adobe Fonts plans to use OpenBakery instead of FontBakery for its font processing pipeline.

Yes, we're planning to switch. The change will happen as soon as the next OpenBakery release is out.

> We (The Type Founders) specifically started using FontBakery for validating our fonts to streamline onboarding fonts on Adobe Fonts. Does this fork imply that we'll need to run our fonts through OpenBakery in the (near?) future?

Yes, `openbakery` will be the tool that we'll start recommending to our foundry partners.

> will the Adobe Fonts profile in FontBakery also be maintained by Adobe?

Our plan is to maintain the Adobe Fonts profile in FontBakery until OpenBakery is ready for general use. Once that happens, we will remove our profile from FontBakery so not to cause confusion in users and not to be the source of extra maintenance for developers.

**Comment by bramstein** [comment]
Hi Miguel! Thanks for clarifying, I understand it is still early. We'll wait for the official announcement from Adobe. Until then, if we find issues in the `adobefonts` profile, should we post them here or in the Fontbakery repository?

**Comment by miguelsousa** [comment]
Please post them in the FontBakery repo for now. Let's keep it the source of truth until the switch happens. I'll copy over the changes. Thanks.
